js/contact.js

```diff
@@ -1,38 +1,55 @@
-const path = require('path');
-const dir = '/home/raimelmedina/Documents/yunet/webserver/';
-  
-
-// create my dir
-/* const dir = '/home/raimelmedina/Documents/yunet/webserver/';
-
-// configure body-parser
-app.use(express.static(path.join(dir)));
-app.use(bodyParser.urlencoded({extended: true}));
-app.use(bodyParser.json()); */
+let queryName = "";
 
 // GET method route
 const getContact = (req, res) => {
-  res.status(200)
-     .sendFile(path.join(dir + '/index.html'));         
+  queryName = req.query.name;
+  if (queryName == undefined) {
+    queryName = "";
+  } 
+  var html =`<form action="/contact-us" method="post"> 
+               Enter your name*: 
+               <br> 
+               <input type="text" name="name" placeholder="Full name" value=${queryName}> 
```

Comment:
This does not work if the name contains spaces, you should put into quotes
```suggestion
               <input type="text" name="name" placeholder="Full name" value="${queryName}"> 
```